### PR TITLE
feat(node): v13.13

### PR DESCRIPTION
**Pull request by SimonSchick**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V13.md#13.13.0
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by SimonSchick** [comment]
Build failure appears to be unrelated to my changes?
https://dev.azure.com/definitelytyped/DefinitelyTyped/_build/results?buildId=41260&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=40b1ee41-44d6-5bba-aa04-4b76a5c732e5&l=57

**Review by Flarna — APPROVED** [review]

File: types/node/worker_threads.d.ts [hunk 1]
```diff
@@ -85,7 +89,12 @@ declare module "worker_threads" {
         readonly threadId: number;
         readonly resourceLimits?: ResourceLimits;
 
-        constructor(filename: string, options?: WorkerOptions);
+        /**
+         * @param filename  The path to the Worker’s main script or module.
+         *                  Must be either an absolute path or a relative path (i.e. relative to the current working directory) starting with ./ or ../,
+         *                  or a WHATWG URL object using file: protocol. If options.eval is true, this is a string containing JavaScript code rather than a path.
+         */
+        constructor(filename: string | URL, options?: WorkerOptions);
```
Comment (Flarna) [inline types/node/worker_threads.d.ts:97]: I think `URL` needs to be imported.

Comment (SimonSchick) [reply types/node/worker_threads.d.ts:97]: We shouldn't have to, it's a global in node, I willl look into that but I feel like we've already had this issue.

Comment (Flarna) [reply types/node/worker_threads.d.ts:98]: it's once again an issue with DOM.
Refs: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/25356
Refs: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/25342

Comment (SimonSchick) [reply types/node/worker_threads.d.ts:98]: It's also that we cannot re-export types as globals...

**Pull request merged**